en/docs/develop/mi-for-vscode/best-practices.md

```diff
@@ -227,13 +227,505 @@ This guide provides best practices for designing, developing, deploying, and mai
+            ```
+
+        === "New Configuration"
+            ```xml
+            <api context="/HelloWorld" name="HelloWorld" xmlns="http://ws.apache.org/ns/synapse">
+                <resource methods="GET">
+                    <inSequence>
+                        <payloadFactory media-type="json" template-type="default">
+                            <format>{"Hello":"World"}</format>
+                        </payloadFactory>
+                        <respond/>
+                    </inSequence>
+                    <faultSequence/>
+                </resource>
+            </api>
+            ```
+
 - [Loopback Mediator]({{base_path}}/reference/mediators/loopback-mediator/) is no longer recommended.
-- Use the [HTTP Connector]({{base_path}}/reference/connectors/http-connector/http-connector-overview/) to invoke external HTTP services instead of the [Call Mediator]({{base_path}}/reference/mediators/call-mediator/) / [Send Mediator]({{base_path}}/reference/mediators/send-mediator/) / [CallOut Mediator]({{base_path}}/reference/mediators/callout-mediator/). The HTTP Connector provides richer configuration options, and better maintainability. Reserve [Call Mediator]({{base_path}}/reference/mediators/call-mediator/) / [Send Mediator]({{base_path}}/reference/mediators/send-mediator/) / [CallOut Mediator]({{base_path}}/reference/mediators/callout-mediator/) for non-HTTP protocols or legacy scenarios where connectors are not applicable.
-- Use the enhanced [ForEach Mediator]({{base_path}}/reference/mediators/foreach-mediator/) for all scenarios previously handled by the Iterate Mediator or older ForEach. It provides improved performance, supports parallel execution, enables external service calls, and offers more flexible configuration.
-- Use the [Scatter Gather Mediator]({{base_path}}/reference/mediators/scatter-gather-mediator/) instead of the [Clone Mediator]({{base_path}}/reference/mediators/clone-mediator/) and the [Aggregate Mediator]({{base_path}}/reference/mediators/aggregate-mediator/).
-- Use the new [PayloadFactory Mediator]({{base_path}}/reference/mediators/payloadfactory-mediator/) which has been enhanced with inline expression support and performance improvements.
+- Use the [HTTP Connector]({{base_path}}/reference/connectors/http-connector/http-connector-overview/) to invoke external HTTP services instead of the [Call Mediator]({{base_path}}/reference/mediators/call-mediator/) / [Send Mediator]({{base_path}}/reference/mediators/send-mediator/) / [CallOut Mediator]({{base_path}}/reference/mediators/callout-mediator/). The HTTP Connector provides richer configuration options, and better maintainability. Reserve [Call Mediator]({{base_path}}/reference/mediators/call-mediator/) / [Send Mediator]({{base_path}}/reference/mediators/send-mediator/) / [CallOut Mediator]({{base_path}}/reference/mediators/callout-mediator/) for non-HTTP protocols (such as VFS, RabbitMQ, JMS etc.) or legacy scenarios where connectors are not applicable.
```

Comment:
` legacy scenarios where connectors are not applicable`.
Can't think of such usecase. If there is no, shall we remove it?

Reply:
For example, the user can use VFS Transport (using call mediator) or the file connector to write something to a file. We are not promoting to use the transport and instead of that promoting the connectors right?. This one is to reflect that.